.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_gem:
+  rf-stylez: ruby/rubocop.yml
```

Comment:
why this change?

Reply:
This is what it was originally :wink: https://github.com/rainforestapp/circlemator/blob/269080cb69351f9f9c5f995a47585f5478b5199d/.rubocop.yml